js/bootstrap-datepicker.js

```diff
@@ -2034,3 +2038,13 @@
 	});
 
 }));
+
+function deprecate(msg) {
```

Comment:
The issue is that `alias` is executed when the datepicker is injected into jQuery.
IMO it should be executed when any of the aliases is called, not before.
I can PR you a fix for alias then you can rebase on it.